chess/ChessGameModel.swift

```diff
@@ -224,6 +259,10 @@ extension ChessGameModel {
         return message == "white" || message == "black"
     }
 
+    private func isValidEndGameMessage(_ message: String) -> Bool {
```

Comment:
can this check be part of message type? Or you are using message type only within view model?

Reply:
removed it